caluma/caluma_workflow/serializers.py

```diff
@@ -297,6 +297,34 @@ class Meta:
         fields = ("id",)
 
 
+class CancelWorkItemSerializer(serializers.ModelSerializer):
+    id = serializers.GlobalIDField()
+
+    def validate(self, data):
+        try:
+            domain_logic.CancelWorkItemLogic.validate_for_cancel(self.instance)
+        except ValidationError as e:
+            raise exceptions.ValidationError(str(e))
```

Comment:
No need to catch the `ValidationError` and then raising the same Exception.

Reply:
It's not the same.. We catch the django exception and throw the proper exception from DRF.

This is the same in all other mutations that use the domain logic and AFAIR we already discussed this in detail..

Reply:
Ah, missed that. Nevermind.